continuous-integration/linux/platform-dependent/fedora-30/setup.sh

```diff
@@ -13,17 +14,46 @@ alternatives --install /usr/bin/python python /usr/bin/python3 0 || exit $?
 alternatives --install /usr/bin/pip pip /usr/bin/pip3 0 || exit $?
 
 function install_opencl_cpu_runtime {
+
+    # Install driver prerequisites
+    dnf --assumeyes install \
+        numactl-libs \
+        redhat-lsb-core \
+        || exit $?
+
     TMP_DIR=$(mktemp --tmpdir --directory zivid-setup-opencl-cpu-XXXX) || exit $?
     pushd $TMP_DIR || exit $?
-    wget -nv https://www.dropbox.com/s/0cvg8fypylgal2m/opencl_runtime_16.1.1_x64_ubuntu_6.4.0.25.tgz || exit $?
-    tar -xf opencl_runtime_16.1.1_x64_ubuntu_6.4.0.25.tgz || exit $?
-    dnf install --assumeyes opencl_runtime_*/rpm/*.rpm || exit $?
+    wget -q https://www.dropbox.com/s/h0txd04aqfluglq/l_opencl_p_18.1.0.015.tgz || exit $?
+    tar -xf l_opencl_p_18.1.0.015.tgz || exit $?
+    cd l_opencl_*/ || exit $?
+
+    cat > installer_config.cfg <<EOF
+# See silent.cfg in the .tgz for description of the options.
+ACCEPT_EULA=accept
+# 'yes' below is required because the installer officially supports only Ubuntu 16.04. However, it will
```

Comment:
Should fix this comment to make sense for fedora and arch also

Reply:
Fixed